docs/application/dotnet/guides/location-sensors/device-sensors.md

```diff
@@ -18,14 +18,14 @@ A device can have various physical and virtual sensors. The following table list
 
 | Supported sensor types                   |                                          |                                       |
 |----------------------------------------|----------------------------------------|-------------------------------------|
-| [Accelerometer](#accelerometer)  | [Humidity Sensor](#humidity)   | [Proximity Sensor](#proximity)   |
-| [AutoRotation Sensor](#auto_rotation)   | [Light Sensor](#light)   | [Rotation Vector Sensor](#rotation)  |
-| [Geomagnetic Rotation Vector Sensor](#mag_rotation)  | [Linear Acceleration Sensor](#lin_accelerometer)  | [Sleep Monitor](#sleep_monitor)  |
-| [Gravity Sensor](#gravity)   | [Magnetic Sensor](#magnetic) | [Temperature Sensor](#temperature)  |
-| [Gyroscope](#gyro)    | [Orientation Sensor](#orientation)  | [Ultraviolet Sensor](#ultraviolet)  |
-| [Gyroscope Rotation Vector Sensor](#gyro_rotation)  | [Pedometer](#pedometer)  | [Uncalibrated Gyroscope](#uncal_gyro)  |
-| [Heart Rate Monitor Sensor](#hrm)   | [Pressure Sensor](#pressure)   | [Uncalibrated Magnetic Sensor](#uncal_magnetic) |
-
+| [Accelerometer](#accelerometer)  | [Heart Rate Monitor LED Green Batch Sensor](#hrm_green_batch) | [Proximity Sensor](#proximity)   |
+| [AutoRotation Sensor](#auto_rotation) | [Humidity Sensor](#humidity) | [Rotation Vector Sensor](#rotation)  |
+| [Geomagnetic Rotation Vector Sensor](#mag_rotation) | [Light Sensor](#light) | [Sleep Monitor](#sleep_monitor)  |
+| [Gravity Sensor](#gravity)   | [Linear Acceleration Sensor](#lin_accelerometer) | [Temperature Sensor](#temperature)  |
+| [Gyroscope](#gyro)    | [Magnetic Sensor](#magnetic) | [Ultraviolet Sensor](#ultraviolet)  |
+| [Gyroscope Rotation Vector Sensor](#gyro_rotation)  | [Orientation Sensor](#orientation) | [Uncalibrated Gyroscope](#uncal_gyro)  |
+| [Heart Rate Monitor Sensor](#hrm)   | [Pedometer](#pedometer) | [Uncalibrated Magnetic Sensor](#uncal_magnetic) |
```

Comment:
@bwikbs Add the new sensors according to alphabetical order.

For reference, kindly check @mijin85cho's comment in PR https://github.com/Samsung/tizen-docs/pull/1115#discussion_r429719639

Reply:
@rishu-samsung  It's same with native api(https://github.com/Samsung/tizen-docs/blob/master/docs/application/native/guides/location-sensors/device-sensors.md). I thought it should be consistent between the dot net and native API documentation.

Reply:
Thanks for the checking @rishu-samsung 

According to the alphabetical order, the items should follow below order.

 - ...
 - Heart Rate Monitor Batch Sensor
 - Heart Rate Monitor LED Green Batch Sensor
 - Heart Rate Monitor Sensor
 - ...

However, in this chase, I think we can consider "Heart Rate Monitor" as the main notion and "Heart Rate Monitor Batch" & "Heart Rate Monitor LED Green Batch" as the related notion to "Heart Rate Monitor". 
So for "Heart Rate Monitor",  "Heart Rate Monitor Batch" and "Heart Rate Monitor LED Green Batch",
I think this order is accepted.

 - ...
 - Heart Rate Monitor Sensor
 - Heart Rate Monitor Batch Sensor
 - Heart Rate Monitor LED Green Batch Sensor
 - ...

Reply:
@mijin85cho Thanks for clarifying the same. We can retain it, as suggested 
I have a doubt. if we keep alphabetical order at other places but not here won't that create confusion for reader/end-user?

Reply:
> @mijin85cho Thanks for clarifying the same. We can retain it, as suggested
> I have a doubt. if we keep alphabetical order at other places but not here won't that create confusion for reader/end-user?

@mijin85cho Please share your opinion for the same.

Reply:
@rishu-samsung Yes, actually exceptions are not suggested while managing the document as it can give confusion and the feeling of inconsistency.

However, as I commented above, in this case I think we can put the "Heart Rate Monitor Sensor" before "Heart Rate Monitor Batch Sensor" and "Heart Rate Monitor LED Green Batch Sensor",
as the "Heart Rate Monitor Sensor" can be considered as the basic notion compared to other two.

Reply:
@mijin85cho Thanks for your opinion. If that is the case, I will go as you suggested.